src/coreclr/vm/peimage.inl

```diff
@@ -385,24 +389,6 @@ inline BOOL PEImage::HasPath()
     return !GetPath().IsEmpty();
 }
```

Comment:
Delete `HasPath`? Is it no longer used and it is inefficient now.

Reply:
Oops. I thought I reverted that. Thanks.